app/routes/goal_routes.py

```diff
@@ -1 +1,57 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response
+from app.models.goal import Goal
+from app.models.task import Task
+from app.db import db
+from app.routes.route_utilities import *
+
+bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+
+@bp.post("/", strict_slashes=False)
+def create_goal():
+    return create_class_instance(Goal, request, ["title"])
```

Comment:
The logic in `create_class_instance` doesn't access the list of required fields so I think `["title"]` should be removed since we don't need to pass it to the helper function